docs/user_guide/tensorflow/model_serving.rst

```diff
@@ -24,73 +22,119 @@ To demonstrate this we provide an example notebook which is also included in the
+        ├── 1
+        │   ├── saved_model.pb
+        │   └── variables
+        │       ├── variables.data-00000-of-00001
+        │       └── variables.index
+        └── 2
+            ├── saved_model.pb
+            └── variables
+                ├── variables.data-00000-of-00001
+                └── variables.index
+
+TensorFlow serving expects the model directory (In this case *mnist*) to contain one or more sub-directories.
+The name of each sub-directory is a number representing the version of the model, the higher the version, the more recent the model.
+Inside each version directory TensorFlow serving expects a file named *saved_mode.pb*, which contains the graph of the model, and a directory called *variables* which contains the weights of the model.
 
 **Step 2.**
 
-The next step is to create a serving definition in the Hopsworks Model Serving service.
+To start serving your model, create a serving definition in the Hopsworks Model Serving service. Clik on the Model Serving service on the left panel (1) and then click on *Create new serving* (2).
    
-.. _model_serving.png: ../../_images/model_serving.png
-.. figure:: ../../imgs/model_serving.png
+.. _serving1.png: ../../_images/serving/serving1.png
+.. figure:: ../../imgs/serving/serving1.png
    :alt: New serving definition
-   :target: `model_serving.png`_
+   :target: `serving1.png`_
    :align: center
+   :width: 400px
+   :height: 400px
    :figclass: align-center
     
+Next click on the model button to select from your project the model you want to serve.
 
-Click the Model button
-        
-.. _serving_definition.png: ../../_images/serving_definition.png
-.. figure:: ../../imgs/serving_definition.png
+.. _serving2.png: ../../_images/serving/serving2.png
+.. figure:: ../../imgs/serving/serving2.png
    :alt: Create serving
-   :target: `serving_definition.png`_
+   :target: `serving2.png`_
    :align: center
    :figclass: align-center
-    
-Select the .pb file in your Models dataset
+
+This will open a popup window that will allow you to browse your project and select directory containing  the mdoel you want to serve. You should select the model directory, meaning the directory containing the sub-directories with the different versions of your model. In the example below we have exported two versions of the *mnist* model. In this step we select the *mnist* directory containing the two versions. 
    
-.. _select_model.png: ../../_images/select_model.png
-.. figure:: ../../imgs/select_model.png
-   :alt: Select protobuf model
-   :target: `select_model.png`_
+.. _serving3.png: ../../_images/serving/serving3.png
+.. figure:: ../../imgs/serving/serving3.png
+   :alt: Select model directory 
+   :target: `serving3.png`_
    :align: center
    :figclass: align-center
     
-Then select batching if it should be used and create your serving.
+After clicking select the popup window close and the information in the create serving menu will be filled in automatically. 
```

Comment:
@SirOibaf `clicking, select`

Reply:
`select` is what's written on the button

Reply:
I'm going to put it italic